sdk/include/sdk/IPubSubClient.h

```diff
@@ -134,6 +155,13 @@ class IPubSubClient {
      */
     virtual AsyncSubscriptionPtr_t<std::string> subscribeTopic(const std::string& topic) = 0;
 
+    /**
+     * @brief Unsubscribe from a topic.
+     *
+     * @param topic   The topic to unsubscribe from.
+     */
+    virtual void unsubscribeTopic(const std::string& topic) = 0;
+
```

Comment:
There could be different subscriptions to the same topic. Therefore we should also offer the option to recall just a specific subscription:
```suggestion

    /**
     * @brief Recall a specific subscription
     *
     * @param subscription   The subscription to recall; this is an object earlier returned by subscibeTopic.
     */
    virtual void unsubscribeTopic(const AsyncSubscriptionPtr_t<std::string>& subscription = {}) = 0;
```